DataFormats/TrackerRecHit2D/interface/VectorHit2D.h

```diff
@@ -0,0 +1,32 @@
+#ifndef TrackerRecHit2D_VectorHit2D_h
+#define TrackerRecHit2D_VectorHit2D_h
+
+#include "DataFormats/GeometryVector/interface/LocalVector.h"
+#include "DataFormats/GeometryVector/interface/LocalPoint.h"
+#include "DataFormats/CLHEP/interface/AlgebraicObjects.h"
+#include "DataFormats/GeometrySurface/interface/LocalError.h"
+
+class VectorHit2D {
```

Comment:
here and other DF header files should have some sensible comments for self-documentation.
This class also looks like a utility class which has no persistency.
Perhaps it can just be moved inside the VectorHit class